src/utils/metrics.ts

```diff
@@ -359,15 +369,26 @@ export async function insertOss(
     data.target_id = pull_request.number;
     data.target_name = review.state;
     data.target_type = 'pull_request';
+  } else if (eventType === 'discussion') {
+    const { discussion } = payload;
+    data.object_id = discussion.number;
+    data.created_at = discussion.created_at;
+    data.updated_at = discussion.updated_at;
+    data.target_id = discussion.number;
+    data.target_name = discussion.title;
+    data.target_type = 'discussion';
+  } else if (eventType === 'discussion_comment') {
+    const { discussion, comment } = payload;
+    data.object_id = discussion.number;
+    data.created_at = comment.created_at;
+    data.updated_at = comment.updated_at;
+    data.target_id = comment.id;
+    data.target_name = discussion.title;
+    data.target_type = 'discussion';
   } else {
     // Unknown payload event, ignoring...
     return {};
   }
-  data.teams = getTeams(
```

Comment:
I think it's wrong to have a team assigned to every GitHub event, especially if we're only tracking it for `issue` and `issue_comment` event types